rebar.OTP18.config

```diff
@@ -1,5 +1,5 @@
 %%-*- mode: erlang -*-
```

Comment:
please let's keep `rebar.config` file as well (maybe having it as a symlink to ` rebar.OTP20.config ` or other way around). I'm afraid it might cause issues when `jesse` is added as a dependency if there is no `rebar.config`